packages/firebase_auth/README.md

```diff
@@ -83,11 +83,30 @@ _handleSignIn()
     .catchError((e) => print(e));
 ```
 
+### Register a user
+
+You can now register a new user using firebase_auth, e.g.
+```dart
+final FirebaseUser user = await _auth.createUserWithEmailAndPassword(
+      email: 'an email',
+      password: 'a password',
+    );
+```
+
+### Supported firebase authentication methods.
+
+Supported authentication methods includes `Google authentication`, `Email&Password authentication`, `Phone authentication` and `Anonymously Sign in`.
```

Comment:
I think we should include the full list if we're going to include a list them at all.

This sentence has some capitalization, spacing, grammar, and consistency issues. Maybe just have a bulleted list instead of a sentence.

Reply:
Do you mean the' full list' that supported by Firebase or supported by us? The list I added here is all the supported I found in dart implementation, maybe I missed something? 
Will fix the english style issues.

Reply:
We also support Twitter, Facebook, and GitHub. See the full list of auth providers here:

https://github.com/flutter/plugins/tree/master/packages/firebase_auth/lib/src/auth_provider